debs/SPECS/wazuh-agent/debian/changelog

```diff
@@ -21,7 +27,7 @@ wazuh-agent (4.4.0-RELEASE) stable; urgency=low
   * More info: https://documentation.wazuh.com/current/release-notes/
 
  -- Wazuh, Inc <info@wazuh.com>  Wed, 18 Jan 2023 12:31:50 +0000
-
+ 
```

Comment:
Remove this space

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/2187/commits/ba90ffbca3b797b495490c5e0e83e004d6b5c304